src/settings/s_decisions.php

```diff
@@ -147,7 +147,80 @@ function render(SettingValues $sv) {
         "<p class=\"settingtext\">To collect <em>multiple</em> final versions, such as one in 9pt and one in 11pt, add “Alternate final version” options via <a href='", hoturl("settings", "group=opt"), "'>Settings &gt; Submission options</a>.</p>",
         "</div>\n\n";
     Ht::stash_script("foldup(\$\$('cbfinal_open'),null,{f:\"c\"})");
+
+	//Share with Open Academic
+    echo "<h3 class=\"settings g\">Share with search engines</h3>\n";
+    echo "<p><b>Click the button to share accepted paper meta data with search engines</b></p>";
+    echo Ht::submit("share", "Share with search engines");
+
+    //wakes up API
+    $api = 'https://openacademicapi.azurewebsites.net/api/papers';
+    $opts = array('http'=> array('method' => 'GET'));
+    $context = stream_context_create($opts);
+    $post_result = file_get_contents($api, false, $context);
```

Comment:
Uh, what's the purpose of this here? What's the $post_result? I get the lower use of file_get_contents/steram_context_create, but not this one.